src/dev/lupio/LupioRTC.py

```diff
@@ -0,0 +1,19 @@
+from m5.params import *
+from m5.proxy import *
+from m5.objects.Device import BasicPioDevice
+from m5.util.fdthelper import *
+
+class LupioRTC(BasicPioDevice):
+    type = 'LupioRTC'
+    cxx_class='gem5::LupioRTC'
+    cxx_header = 'dev/lupio/lupio_rtc.hh'
+    time = Param.Time('01/01/2012',
```

Comment:
Is there a reason behind this date? Or is it just a copy from another file? What about the typical 01/01/1970 otherwise?

Reply:
How about 1/1/2020 for gem5-20 (kinda)?

We probably don't want to use 1/1/1970 in case some installed program cares about the time. It may be weird to see the time be so old.

Reply:
Funnily enough, I believe that raspberryPi don't have an RTC, so if they boot for the first time without network access to set up NTP, they actually show Jan 1st 1970!